addons/hr/hr_tip_data.xml

```diff
@@ -2,32 +2,31 @@
 <openerp>
     <data>
         <record model="web.tip" id="hr_tip_1">
-            <field name="title"></field>
-            <field name="description">By clicking here, you can keep a track of yours applications concerning this job position. You'll be able to plan meetings and manage documents related to the applicants.</field>
-            <field name="action_id" ref="hr.action_hr_job"/>
+            <field name="title">Awesome</field>
+            <field name="description">Here is your job position. Click here to track your applications and plan meetings with them. Let the recruitment begin !</field>
             <field name="model">hr.job</field>
             <field name="mode">form</field>
             <field name="trigger_selector">.oe_form_buttons_view:visible,div.oe_right > .oe_button_box > button</field>
-            <field name="highlight_selector">div.oe_right > .oe_button_box:visible > button:nth-child(1)</field>
+            <field name="highlight_selector">div.oe_right > .oe_button_box:visible > #total_application</field>
             <field name="placement">bottom</field>
         </record>
 
         <record model="web.tip" id="hr_tip_2">
-            <field name="title"></field>
-            <field name="description">This is where you navigate through your process. Click here to go back to where you came from.</field>
+            <field name="title">Great</field>
+            <field name="description">Here is your applicant information. You can plan and start an interview. If it is positif, don't forget to create your new employee !</field>
             <field name="model">hr.applicant</field>
             <field name="mode">form</field>
-            <field name="trigger_selector">ol.breadcrumb:visible</field>
-            <field name="highlight_selector">ol.breadcrumb a:nth-child(3)</field>
-            <field name="placement">bottom</field>
+            <field name="trigger_selector">.oe_form_buttons_view:visible,div.oe_right.oe_button_box > button</field>
+            <field name="highlight_selector">.oe_form_sheet:visible</field>
+            <field name="placement">auto top</field>
         </record>
 
         <record model="web.tip" id="hr_tip_3">
-            <field name="title"></field>
-            <field name="description">Congratulations! Your employee has been created. In this form, you can modify his personal information or even add a picture.</field>
+            <field name="title">Your employee</field>
+            <field name="description">Here is the employee information. Modify his personal information or even add a picture.</field>
```

Comment:
Can't we check the action of the employee creation from the application form, in order to trigger the tip if and only if we're coming from that action ?

Here, the tip is triggered, even if I come from the Leaves menu, or the expense menu...